### PR TITLE
DIV-2649

### PR DESCRIPTION
**Pull request by niyioyelade**
Special characters in names - Fix for Special Character Ł not showing up in Mini Petition

# Description
https://tools.hmcts.net/jira/browse/DIV-2649

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/integrationTest/resources/documentgenerator/d8minipetition/jsoninput/CC--cores_FO--ChildApp_CN--BCDE_DR-AD-CRK-Yes-PL-Yes-DT-Yes_LP--Yes.json [hunk 1]
```diff
@@ -150,7 +150,7 @@
         "D8RespondentCorrespondenceAddress": {
           "PostCode": "SW9 9PE"
         },
-        "D8DerivedPetitionerCurrentFullName": "John Smith",
+        "D8DerivedPetitionerCurrentFullName": "Łukasz John Smith",
```
Comment (mthava) [inline src/integrationTest/resources/documentgenerator/d8minipetition/jsoninput/CC--cores_FO--ChildApp_CN--BCDE_DR-AD-CRK-Yes-PL-Yes-DT-Yes_LP--Yes.json]: Can you add other characters as well.

Comment (niyioyelade) [reply src/integrationTest/resources/documentgenerator/d8minipetition/jsoninput/CC--cores_FO--ChildApp_CN--BCDE_DR-AD-CRK-Yes-PL-Yes-DT-Yes_LP--Yes.json]: sure

Comment (niyioyelade) [reply src/integrationTest/resources/documentgenerator/d8minipetition/jsoninput/CC--cores_FO--ChildApp_CN--BCDE_DR-AD-CRK-Yes-PL-Yes-DT-Yes_LP--Yes.json]: Done

**Comment by nsingamsetty** [comment]
This is verified with the attached PDF for the ticket DIV02649 looks good.  PR build is successful and BA approved it - Refer the JIRA for more details hence merging the change

**Pull request merged**